elastic/kibana
# Switching to less repo

## Summary

Switching to original `less` repo. 

## Testing

1. Pull this PR locally and start KB
2. Add the Sample Data
3. Navigate to the Visualize Library -> Create New Visualization -> Choose TSVB
4. Click the Markdown tab, then the Panel Options tab below
5.  In the editor add the following: `@plugin "https://www.example.com/plugin";`
6. Open your browsers Network tool
7. Alter the plugin link, notice that it is not trying to retrieve the resource js file
8. In `src/plugins/vis_types/timeseries/public/application/components/panel_config/markdown.tsx` change the `disablePluginRule` to `false`
9. Repeat steps 5-7.
10. Notice in the Network tool that you see Kibana trying to retrieve the js file


# Screenshots

`disablePluginRule` to `false`
<img width="1146" alt="Screenshot 2023-10-17 at 5 49 29 PM" src="https://github.com/elastic/kibana/assets/21210601/237f8287-87b6-43de-af23-1b21eae7cccd">


`disablePluginRule` to `true`
<img width="1030" alt="Screenshot 2023-10-18 at 10 28 06 AM" src="https://github.com/elastic/kibana/assets/21210601/3b92dc3c-f8c7-4b74-b0d9-602c67c25d6c">
